lib/src/js/debug/src/standard/gestures.html

```diff
@@ -404,6 +403,20 @@
       if (recognizer.info) {
         recognizer.info.prevent = true;
       }
+    },
+    /**
+     * Reset the 2500ms timeout on processing mouse input after detecting touch input.
+     *
+     * Touch inputs create synthesized mouse inputs anywhere from 0 to 2000ms after the touch.
+     * This method should only be called during testing with simulated touch inputs.
+     * Calling this method in production may cause duplicate taps or other gestures.
+     *
+     * @method resetMouseCanceller
+     */
+    resetMouseCanceller: function() {
```

Comment:
I am not sure if this is actually exposed publicly, looks like it ends up on `Polymer.Gestures`. Should we possibly have a PolymerGestures class in Dart with some static methods? Probably not necessary as part of this pull request, but might be a good thing to add eventually.